web/src/app/shared/header/header.component.html

```diff
@@ -80,7 +80,7 @@
                     </ul>
                 </li>
                 <li>
-                    <span class="name-img">VK</span>
+                    <span class="name-img">{{firstName}}{{lastName}}</span>
```

Comment:
Here initial need to display, i.e first character of  first name and last name and combine it in single variable.

Reply:
Done